test/zzzz-duplicate-entries.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# This is a test suite for testing basic log processing
+# functionality and existance of duplicate records when journal rolls over
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+os::util::environment::use_sudo
```

Comment:
I think this test should only run if `log-driver=journald` - the `add_test_message` function will add messages to kibana, and works by having fluentd get the kibana logs.  If using `log-driver=json-file` the logs will go to the json files, not the journal.

Reply:
is there a way i can restrict which this runs against? or is there a clear and easy way for me to push messages straight to journal?

Reply:
There isn't really a detriment if it runs against both though I suppose... it just doesn't test what we're trying to test otherwise...